src/org/testKitGen/TestInfoParser.java

```diff
@@ -190,14 +190,17 @@ private void parseDisableInfo(TestInfo ti) {
 			String impl = getDisabledEle(disabled, "impl", ti.getTestCaseName());
 			String vendor = getDisabledEle(disabled, "vendor", ti.getTestCaseName());
 			String version = getDisabledEle(disabled, "version", ti.getTestCaseName());
-			String plat = getDisabledEle(disabled, "plat", ti.getTestCaseName());
+			String platform = getDisabledEle(disabled, "platform", ti.getTestCaseName());
+			if (platform != null) {
+				platform = getDisabledEle(disabled, "plat", ti.getTestCaseName());
```

Comment:
Is that to support backwards compatibility?

Reply:
Thanks for catching it. I mistakenly had `!=` instead of `==`, updated the change.